test/agent.ts

```diff
@@ -14,11 +14,11 @@ test.before('setup', async () => {
 
 	// HTTPS Handlers
 
-	https.on('/', (request, response) => {
+	https.on('/', (_unusedRequest, response) => {
```

Comment:
I think just `_request` would be better.

Reply:
Sure, I'll change it if you'd like. I went with this because with `_request`, it seems likely that if in the future the code were changed and _did_ start to use the parameter, the programmer likely would not know to remove the underscore. Whereas if you start calling things on `_unusedRequest` it screams that the name should be changed.